### PR TITLE
fix: code and localization

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: src/ResourceList.js [hunk 1]
```diff
@@ -30,268 +32,149 @@ class ResourceList extends H5P.EventDispatcher {
...
-    this.on('resize', () => {
-      this.resize.bind(this);
     });
+    this.currentRatio = ratio;
+  }
+
+  /**
+   * Get list of classname and conditions for when to add the classname to the content type
+   * 
+   * @return {[{className: string, shouldAdd: (function(*): boolean)}, {className: string, shouldAdd: (function(*): boolean|boolean)}, {className: string, shouldAdd: (function(*): boolean)}]}
```
Comment (boyum) [inline src/ResourceList.js:161]:
```suggestion
   * @return {{ className: string, shouldAdd: (ratio: number) => boolean}[]}
```

File: src/ResourceList.js [hunk 2]
```diff
@@ -30,268 +32,149 @@ class ResourceList extends H5P.EventDispatcher {
...
+        'className': this.mediumTabletSurface,
+        'shouldAdd': (ratio) => ratio >= 22 && ratio < 42,
+      },
+      {
+        'className': this.largeTabletSurface,
+        'shouldAdd': (ratio) => ratio >= 42 && ratio < 60,
+      },
+      {
+        'className': this.largeSurface,
+        'shouldAdd': (ratio) => ratio >= 60,
```
Comment (boyum) [inline src/ResourceList.js:175]:
```suggestion
        className: this.mediumTabletSurface,
        shouldAdd: (ratio) => ratio >= 22 && ratio < 42,
      },
      {
        className: this.largeTabletSurface,
        shouldAdd: (ratio) => ratio >= 42 && ratio < 60,
      },
      {
        className: this.largeSurface,
        shouldAdd: (ratio) => ratio >= 60,
```

File: src/ResourceList.js (hunk 1 above)
Comment (henriettemoe) [reply src/ResourceList.js:161]: Nice!

File: src/ResourceList.js [hunk 3]
```diff
@@ -100,266 +29,152 @@ H5P.ResourceList = (function () {
...
+      this.listContainer.classList.remove('hidden');
+      this.listContainer.classList.remove('slide-out');
+      this.listContainer.classList.add('slide-in');
+    }
+    else {
+      this.wrapper.onkeydown = () => { };
+      this.listContainer.classList.remove('slide-in');
+      this.listContainer.classList.add('slide-out');
+      setTimeout(() => this.listContainer.classList.add('hidden'), 500);
+    }
```
Comment (boyum) [inline src/ResourceList.js:123]: Would be nice to swap this if/else.

```suggestion
    if (!isActive) {
      this.wrapper.onkeydown = () => { };
      this.listContainer.classList.remove('slide-in');
      this.listContainer.classList.add('slide-out');
      setTimeout(() => this.listContainer.classList.add('hidden'), 500);
    }
    else {
      const focusableElements = Array.from(this.listContainer.querySelectorAll(focusableElementsString));
      this.wrapper.onkeydown = (event) => trapKeys(event, focusableElements[0], focusableElements[focusableElements.length - 1], this.toggleResources);
      this.listContainer.classList.remove('hidden');
      this.listContainer.classList.remove('slide-out');
      this.listContainer.classList.add('slide-in');
    }
```

**Review by boyum — APPROVED** [review]
✨

File: src/ResourceList.js (hunk 3 above)
Comment (henriettemoe) [reply src/ResourceList.js:123]: Yess

**Pull request merged**